README.md

```diff
@@ -162,19 +200,21 @@ You can then submit the script using the command:
 ```
 qsub runPipeline_script.sh
 ```
-**Note**
-- The main script is excecuted using the queue `copyq` so that the singularity container images required by the pipeline are downloaded.
-- The NCI Gadi config currently runs all tasks on the normal queue. This config uses the `--gadi-account` flag to assign a project code to all task job submissions for billing purposes. The version of Nextflow installed on Gadi has been modified to make it easier to specify resource options for jobs submitted to the cluster. See NCI's [Gadi user guide](https://opus.nci.org.au/display/DAE/Nextflow) for more details.
-
 
 ### 5. Results 
 Once the pipeline is complete, you will find all outputs for each sample in the `results` directory. 
 
 ## Infrastructure usage and recommendations
```

Comment:
See comments above about clearly capturing execution information for Gadi in one place